### PR TITLE
feat(firstLetters): add `withVishraams` option

### PR DESCRIPTION
**Pull request by Harjot1Singh**
Adds the option to retain vishraams with `firstLetters`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +88,11 @@ firstLetters('iZir&qym sMdUk drIXw AmIk ]') // => gsdA
 ```js
 firstLetters('iZir&qym sMdUk* drIXw AmIk* ]', false) // => Zsda
 ```
-### toAscii(text) ⇒ <code>String</code>
+**Example** *(ASCII first letters with vishraams)*  
+```js
+firstLetters('iZir&qym sMdUk* drIXw AmIk* ]', true, true) // => Zsda
```
Comment (bhajneet) [inline README.md]: Not the best example lol

Comment (Harjot1Singh) [reply README.md]: Oh, copied it and forgot to change it lol

Comment (Harjot1Singh) [reply README.md]: Done

**Comment by Harjot1Singh** [comment]
Changes done

**Pull request merged**